Sprint 2/ClearScreen.java

```diff
@@ -0,0 +1,10 @@
+import java.io.IOException;

```

Comment:
For all of your code, I think that you should provide the author of the code and a short description of what the code does at the top of the page.